### PR TITLE
feat(ui5-application-inquirer): Adds `defaultValue` opts for some prompt

### PR DESCRIPTION
**Pull request by IainSAP**
https://github.com/SAP/open-ux-tools/issues/2277

- Adds new prompt option `defaultValue` to `name` and `targetFolder` prompts
- Adds tests to cover
- Existing behaviour is maintained

### CODE REVIEW

File: packages/inquirer-common/jest.config.js [hunk 1]
```diff
@@ -3,4 +3,5 @@ config.snapshotFormat = {
     escapeString: false,
     printBasicPrototype: false
 };
+config.collectCoverage = false;
```
Comment (devinea) [inline packages/inquirer-common/jest.config.js:6]: revert?

**Review by mikicvi-SAP — APPROVED** [review]
Changeset ✅
- Tested locally, working as expected - existing behavior still supported, new functionality fixes the issue (internal #30256).
- Code is easy to read follow
- 100% coverage & unit tests covering the change

**Review by cianmSAP — APPROVED** [review]
Updates are clear
Tests to cover
I have not tested this locally

**Review by devinea — APPROVED** [review]
Thanks for updates. lg

**Pull request merged**